src/AccessibilityInsights.SharedUx/Controls/CustomControls/ScannerResultCustomListControl.xaml.cs

```diff
@@ -0,0 +1,266 @@
+            {
+                ListView listView = sender as ListView;
+                GridView gView = listView.View as GridView;
+
+                if (gView.Columns.Count >= 2)
+                {
+                    var width = (listView.ActualWidth - SystemParameters.VerticalScrollBarWidth) - gView.Columns[1].ActualWidth;
+                    //leave the width as it was, if the resulting width goes in negative.
+                    if (width >= 0)
+                    {
+                        gView.Columns[0].Width = width;
+                    }
+                }
+            }
+        }
+
+        /// <summary>
+        /// Update control based on failure selection
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void lvDetails_SelectedItemChanged(object sender, SelectionChangedEventArgs e)
+        {
+            _controlContext.DataContext.DataContext = (e.AddedItems.Count > 0) ? (ScanListViewItemViewModel)e.AddedItems[0] : null;
+        }
+
+        /// <summary>
+        /// Pass scrolling events through listview
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void lvDetails_PreviewMouseWheel(object sender, MouseWheelEventArgs e)
+        {
+            if (!e.Handled)
+            {
+                e.Handled = true;
+                var eventArg = new MouseWheelEventArgs(e.MouseDevice, e.Timestamp, e.Delta);
+                eventArg.RoutedEvent = MouseWheelEvent;
+                eventArg.Source = sender;
+                var parent = ((Control)sender).Parent as UIElement;
+                parent.RaiseEvent(eventArg);
+            }
+        }
+
+        /// <summary>
+        /// Ensure listview contents are displayed properly
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void lvDetails_IsVisibleChanged(object sender, DependencyPropertyChangedEventArgs e)
+#pragma warning restore CA1801 // unused parameter
+        {
+            if ((bool)e.NewValue)
+            {
+                _controlContext.changeVisibility();
+            }
+        }
+
+        /// <summary>
+        /// Navigate to link on enter in listview
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void ListViewItem_KeyDown(object sender, KeyEventArgs e)
+        {
+            if (e.Key == Key.Enter)
+            {
+                ((sender as ListViewItem).DataContext as ScanListViewItemViewModel).InvokeHelpLink();
+                e.Handled = true;
+            }
+        }
+
+        /// <summary>
+        /// Handles click on file bug button
+        /// </summary>
+        /// <param name="sender"></param>
+        /// <param name="e"></param>
+        private void btnFileBug_Click(object sender, RoutedEventArgs e)
+        {
```

Comment:
I just opened https://mseng.visualstudio.com/1ES/_workitems/edit/1959216 to have something to track the conversation that we've had about refactoring the common code between here and AutomatedChecksCustomListControl

Reply:
Thanks Dave, I'll left this as it currently works, but I'll make sure to update if we ended up refactoring the bug file code.

Reply:
Agreed. One change at a time